src/test/java/com/github/daniel_sc/rocketchat/modern_client/RocketChatClientIT.java

```diff
@@ -131,10 +133,41 @@ public void testStreamMessages() throws InterruptedException {
                 .join();
 
         ChatMessage receivedMsg = msgStream.blockingFirst();
-        streamDispose.dispose();
 
         assertNotNull(receivedMsg);
         assertEquals(msgText, receivedMsg.msg);
+        assertNull(receivedMsg.editedBy);
+        assertNull(receivedMsg.editedAt);
+        assertFalse(receivedMsg.isModified());
+
+        String msgTextModified = "TEST modern sdk: stream input (modified)";
+        subscription.thenCompose(room -> client.updateMessage(msgTextModified, receivedMsg._id))
+                .join();
+
+        ChatMessage receivedModifiedMsg = msgStream.skip(1).blockingFirst();
+
+        assertNotNull(receivedModifiedMsg);
+        assertEquals(msgTextModified, receivedModifiedMsg.msg);
+        assertNotNull(receivedModifiedMsg.editedBy);
+        assertNotNull(receivedModifiedMsg.editedAt);
+        assertTrue(receivedModifiedMsg.isModified());
+
+        RestApi api = new RestApi(WEB_URL);
```

Comment:
why do you need to test this with the rest api as well? would it not suffice to test it only with the given websocket above? this would keep the test code clean. (If there is some real reason, of course this is good)

Reply:
Unfortunately the live chat API doesn't support adding reactions to messages so REST is the only way to verify that part of the change.

Reply:
I've removed the REST client and the related part of the test